opentelemetry-sdk/src/metrics/instrument.rs

```diff
@@ -218,52 +217,57 @@ impl Stream {
     }
 }
 
-/// the identifying properties of a stream.
+/// The identifying properties of a instrument.
 #[derive(Debug, PartialEq, Eq, Hash)]
-pub(crate) struct StreamId {
+pub(crate) struct InstId {
     /// The human-readable identifier of the stream.
     pub(crate) name: Cow<'static, str>,
     /// Describes the purpose of the data.
     pub(crate) description: Cow<'static, str>,
+    /// Defines the functional group of the instrument.
+    pub(crate) kind: InstrumentKind,
     /// the unit of measurement recorded.
     pub(crate) unit: Unit,
-    /// The stream uses for an instrument.
-    pub(crate) aggregation: String,
-    /// Monotonic is the monotonicity of an instruments data type. This field is
-    /// not used for all data types, so a zero value needs to be understood in the
-    /// context of Aggregation.
-    pub(crate) monotonic: bool,
-    /// Temporality is the temporality of a stream's data type. This field is
-    /// not used by some data types.
-    pub(crate) temporality: Option<Temporality>,
     /// Number is the number type of the stream.
```

Comment:
another `stream` reference above which should be `instrument`.  (trying to be helpful, not picky. ;))

Reply:
thanks